CMakeLists.txt

```diff
@@ -269,6 +272,9 @@ find_package(Boost REQUIRED COMPONENTS filesystem system thread iostreams)
 set(OPENRAVE_BOOST_INCLUDE_DIRS ${Boost_INCLUDE_DIRS})
 set(OPENRAVE_BOOST_LIB_DIRS ${Boost_LIBRARY_DIRS})
 
+# prevents CLANG from "error: integer value -1 is outside the valid range of values [0, 3] for the enumeration type 'sign_mixture_enum' [-Wenum-constexpr-conversion]"
+include_directories(SYSTEM ${OPENRAVE_BOOST_INCLUDE_DIRS})

```

Comment:
@lazydroid do you still need this?

Reply:
@yoshikikanemoto it's not strictly required for suppressing `-Wenum-constexpr-conversion` errors, however, it should help to reduce warning noise coming from Boost and maybe other system includes.

If you think this should be removed, I shall remove it.

Reply:
@yoshikikanemoto this part is gone.

Reply:
ok. thanks